static_frame/core/index_hierarchy.py

```diff
@@ -1051,14 +1052,70 @@ def gen() -> tp.Iterator[tp.Tuple[tp.Hashable, ...]]:
                 )
 
 
-    # def relabel_at_depth(self: IH,
-    #         mapper: RelabelInput,
-    #         depth_level: DepthLevelSpecifier = 0
-    #         ) -> IH:
-    #     '''
-    #     Return a new :obj:`IndexHierarchy` after applying the hte `mapper` to the depth level or levels specified by `depth_level`.
-    #     '''
-    #     pass
+    def relabel_at_depth(self,
+            mapper: RelabelInput,
+            depth_level: DepthLevelSpecifier = 0
+            ) -> "IndexHierarchy":
+        '''
+        Return a new :obj:`IndexHierarchy` after applying `mapper` to a level or each individual level specified by `depth_level`.
+
+        `mapper` can be a callable, mapping, or iterable.
+            - If a callable, it must accept a single value, and return a single value.
+            - If a mapping, it must map a single value to a single value.
+            - If a iterable, it must be the same length as `self`.
```

Comment:
I am not sure if this list presentation translates well in the RST docs... do you know? If not, we can just make it a sentence.

Reply:
I just did a fresh doc build, and it renders nicely, so I will leave it as is :+1: